### PR TITLE
vp8dec: enable svc-t decoding

### PR DESCRIPTION
**Pull request by wudping**
Supporting svc-t decodeing for vp8 with max-layer 3.

Signed-off-by: wudping <dongpingx.wu@intel.com>

### CODE REVIEW

**Comment by wudping** [comment]
please help to review.

**Comment by uartie** [comment]
Depends on 01org/libyami#737

File: tests/decode.cpp [hunk 1]
```diff
@@ -41,8 +41,11 @@ SharedPtr<VppInput> createInput(DecodeParameter& para, SharedPtr<NativeDisplay>&
             return input;
     }else{
         SharedPtr<VppInputDecode> inputDecode = DynamicPointerCast<VppInputDecode>(input);
-        if (inputDecode && inputDecode->config(*display))
-            return input;
+        if (inputDecode) {
+            inputDecode->setSVCTLayer(para.temporalLayer);
```
Comment (xuguangxin) [inline tests/decode.cpp]: Please change to setTargetTemporalLayer.

File: tests/decodehelp.cpp [hunk 2]
```diff
@@ -52,6 +52,9 @@ static void printHelp(const char* app)
     printf("      6: use external dma buf for decode and display, only v4l2decode support this\n");
     printf(" [*] v4l2decode doesn't support the option\n");
     printf("  --capi: use the codec capi to encode or decode, default(false)\n");
+    printf("  --temporal-layer: decode SVC-T stream up to given layer, default 0\n");
```
Comment (xuguangxin) [inline tests/decodehelp.cpp]: Please comment, only vp8 support it.

File: tests/decodehelp.cpp [hunk 3]
```diff
@@ -127,6 +135,11 @@ bool processCmdLine(int argc, char** argv, DecodeParameter* parameters)
         return false;
     }
 
+    if (parameters->temporalLayer > 3) {
```
Comment (xuguangxin) [inline tests/decodehelp.cpp]: No need this, let libyami core decide it

**Comment by wudping** [comment]
updated, thanks.

File: tests/vppinputdecode.h [hunk 4]
```diff
@@ -32,13 +32,20 @@ class VppInputDecode : public VppInput
     bool read(SharedPtr<VideoFrame>& frame);
 
     bool config(NativeDisplay& nativeDisplay);
+    void setTargetTemporalLayer(uint32_t temporalLayer = 0) { m_temporalLayer = temporalLayer; }
```
Comment (xuguangxin) [inline tests/vppinputdecode.h]: Please use one function setTargetLayer(uint32_t temporal, uint32_t spatial,  uint32_t quality)

**Comment by wudping** [comment]
Updated, Thanks @xuguangxin .

**Pull request merged**